templates/macros/filters/dates.html

```diff
@@ -0,0 +1,20 @@
+{% macro field(name, label, dates) %}
```

Comment:
Is this file the same as macros/filters/date.html? Whats the reason for having both?